Description: Add package requirements 
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 yaramanager (1.1-1ubuntu11) trusty; urgency=medium
 .
   * delete unnecessary files, add required packages
Author: Brett Cunningham <brettcu@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- yaramanager-1.1.orig/source/hipara/hipara/settings.py
+++ yaramanager-1.1/source/hipara/hipara/settings.py
@@ -66,7 +66,7 @@ DATABASES = {
     },
     'default': {
         'ENGINE': 'django.db.backends.mysql',
-        'NAME': 'hipara',
+        'NAME': 'mysql',
         'USER': 'bitroots',
         'PASSWORD': 'bitroots',
         'HOST': 'localhost',
@@ -103,7 +103,7 @@ STATICFILES_FINDERS = (
 EMAIL_BACKEND = "djrill.mail.backends.djrill.DjrillBackend"
 MANDRILL_API_KEY = "NdSCtKw7lEn77e9dtvfEQw"
 
-##use following SMTP for PRODUCTION 
+##use following SMTP for PRODUCTION
 # EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
 # DEFAULT_FROM_EMAIL = 'no-reply@hipara.org'
 # SERVER_EMAIL = 'brettcu@gmail.com'
--- yaramanager-1.1.orig/source/hipara/static/dist/css/style.css
+++ yaramanager-1.1/source/hipara/static/dist/css/style.css
@@ -103,7 +103,7 @@ input[type="file"] {
   background-repeat: no-repeat;
   background-size: cover;
   background-position: center;
-  height: 70vh;
+  height: 100vh;
   text-align: center;
 }
 .info_sections3 h1 {
@@ -124,17 +124,22 @@ input[type="file"] {
   border-radius: 5px;
 }
 .info_sections3 > div:nth-of-type(1) {
-  background: #000 none repeat scroll 0 0;
+  /*background: #000 none repeat scroll 0 0;*/
+  background: #4495C6 none repeat scroll 0 0;
   height: 100%;
-  opacity: 0.75;
+  /*opacity: 0.75;*/
+  opacity: 1;
   position: absolute;
   top: 0;
   width: 100%;
 }
 .info_sections3 .row > div {
-  padding: 5em 2em;
+  padding: 2em;
   margin: 0 auto;
 }
+
+.info_sections3 i, .info_sections3 a { color: #fff; cursor: pointer;}
+.info_sections3 a { margin-right: 1em; cursor: pointer; }
 @media screen and (max-width: 64.0625em) {
   .info_sections3 {
     height: 80vh;
--- /dev/null
+++ yaramanager-1.1/source/hipara/static/downloads/deployment_guide.txt
@@ -0,0 +1,137 @@
+sudo apt-get update
+sudo apt-get install git
+sudo apt-get install nginx
+sudo apt-get install mysql-server libmysqlclient-dev    //current password is root
+sudo mysql_install_db
+sudo mysql_secure_installation    //You'll be asked for the administrative password you set for MySQL during installation. Afterwards, you'll be asked a series of questions. Besides the first question which asks you to choose another administrative password, select yes for each question.
+
+create database name 'hipara'
+
+sudo apt-get install python3-pip
+sudo pip3 install virtualenv virtualenvwrapper
+
+echo "export VIRTUALENVWRAPPER_PYTHON=/usr/bin/python3" >> ~/.bashrc
+echo "export WORKON_HOME=~/Env" >> ~/.bashrc
+echo "source '/usr/local/bin/virtualenvwrapper.sh'" >> ~/.bashrc
+source ~/.bashrc
+mkvirtualenv hipara
+
+cd /var/www/html
+
+sudo git clone https://github.com/jbc22/YaraManager.git
+
+cd /var/www/html/YaraManager/source/
+
+pip install -r requirements.txt
+
+sudo nano /var/www/html/YaraManager/source/hipara/hipara/settings.py
+
+	Update :
+	
+	DATABASES = {
+	    'sqlite': {
+		'ENGINE': 'django.db.backends.sqlite3',
+		'NAME': os.path.join(BASE_DIR, 'hipara.sqlite3'),
+	    },
+	    'default': {
+		'ENGINE': 'django.db.backends.mysql',
+		'NAME': 'hipara',
+		'USER': 'root',
+		'PASSWORD': 'root',
+		'HOST': 'localhost',
+		'PORT': '',
+	    }
+	}
+	
+	#Comment following which is test SMTP or leave as it is for development but it will stop working after some time
+	
+	#EMAIL_BACKEND = "djrill.mail.backends.djrill.DjrillBackend"
+	#MANDRILL_API_KEY = "sdgdghdrhdrhbdhrhdrh"
+
+	#Uncomment following and use proper smtp settings for production
+	EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+	DEFAULT_FROM_EMAIL = 'no-reply@hipara.org'   	#user proper email
+	SERVER_EMAIL = 'brettcu@gmail.com'            	#place any email to report site failure
+	EMAIL_HOST = '<EMAIL_HOST>'			#host of smtp service (eg. google)
+	EMAIL_PORT = <PORT>         			#port
+	EMAIL_HOST_USER = '<HOST_EMAIL>'		#host username
+	EMAIL_HOST_PASSWORD = '<HOST_EMAIL_PASSWORD>'  	#host password
+
+
+	#UPDATE Timezone currntly it is UTC
+	
+sudo chmod -R 777 /var/www/html/YaraManager/source/hipara
+python manage.py migrate
+python manage.py collectstatic
+
+deactivate
+
+sudo apt-get install python-dev
+sudo pip3 install uwsgi 		//if uwsgi installed previously then remove it and install with python3
+
+sudo mkdir -p /etc/uwsgi/sites
+cd /etc/uwsgi/sites
+sudo nano hipara.ini
+
+	add following content
+	
+	[uwsgi]
+	project = hipara
+	base = /var/www/html/YaraManager/source
+
+	chdir = %(base)/%(project)
+	home = /home/user/Env/%(project)
+	module = %(project).wsgi:application
+
+	master = true
+	processes = 5
+
+	socket = %(base)/%(project)/%(project).sock
+	chmod-socket = 664
+	vacuum = true
+	
+sudo nano /etc/init/uwsgi.conf
+	add following content in it
+	
+	description "uWSGI application server in Emperor mode"
+
+	start on runlevel [2345]
+	stop on runlevel [!2345]
+
+	setuid www-data
+	setgid www-data
+
+	exec /usr/local/bin/uwsgi --emperor /etc/uwsgi/sites
+
+sudo nano /etc/nginx/sites-available/hipara
+
+	add following content in it
+	
+	server {
+	    listen 80;
+	    server_name hipara.org www.hipara.org;
+
+	    access_log /var/www/html/YaraManager/source/hipara/hipara_access.log;
+    	    error_log  /var/www/html/YaraManager/source/hipara/hipara_error.log;
+		    
+	    location = /favicon.ico { access_log off; log_not_found off; }
+		    
+	    location /static/ {
+		root /var/www/html/YaraManager/source/hipara;
+	    }
+	    location /media/ {
+		root /var/www/html/YaraManager/source/hipara;
+	    }
+
+	    location / {
+		include         /etc/nginx/uwsgi_params;
+		uwsgi_pass      unix:/var/www/html/YaraManager/source/hipara/hipara.sock;
+	    }
+	}
+	
+sudo ln -s /etc/nginx/sites-available/hipara /etc/nginx/sites-enabled
+
+sudo service nginx configtest
+sudo service nginx restart
+sudo service uwsgi restart
+
--- yaramanager-1.1.orig/source/hipara/templates/about.html
+++ yaramanager-1.1/source/hipara/templates/about.html
@@ -15,7 +15,8 @@
             </ol>
         </section>
     </div>
-    <section id="home" class="info_sections3" style="background-image:url({% static page.content.top_banner.image.path %})" alt="{{ page.content.top_banner.image.alt }}" title="{{ page.content.top_banner.image.title }}">
+    <section id="home" class="info_sections3" title="{{ page.content.top_banner.image.title }}">
+        <!-- style="background-image:url({% static page.content.top_banner.image.path %})" alt="{{ page.content.top_banner.image.alt }}" -->
         <div></div>
         <div class="row">
             <div class="columns col-lg-8 col-sm-10 text-center centered">
--- yaramanager-1.1.orig/source/hipara/templates/base.html
+++ yaramanager-1.1/source/hipara/templates/base.html
@@ -2,9 +2,9 @@
 <!DOCTYPE html>
 <html>
 
-<head> 
+<head>
     <title>Hipara | {{ page.title }}</title>
-    
+
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" name="viewport">
@@ -62,7 +62,8 @@
                             </li>
                             <li {% if request.path = '/about/' %} class="active" {% endif %}>
                                 <a href="/about/">
-                                    <i class="glyphicon glyphicon-play"></i>
+                                    <!-- <i class="glyphicon glyphicon-play"></i> -->
+                                    <i class="fa fa-group"></i>
                                     <span>About</span>
                                 </a>
                             </li>
@@ -76,7 +77,7 @@
                             <li class="dropdown {% if request.path = '/export/' or  request.path = '/import/' or '/rule/' in request.path %}active{% endif %}">
                                 <a href="#" class="dropdown-toggle" data-toggle="dropdown">Rules <span class="caret"></span></a>
                                 <ul class="dropdown-menu" role="menu">
-                                    <li {% if request.path = '/export/' %} class="active" {% endif %}><a href="/export/">Extract</a></li>
+                                    <li {% if request.path = '/export/' %} class="active" {% endif %}><a href="/export/">View all rules</a></li>
                                     <li {% if request.path = '/import/' %} class="active" {% endif %}><a href="/import/">Upload</a></li>
                                 </ul>
                             </li>
@@ -126,27 +127,28 @@
                             </li>
                             {% else %}
                             <li class="dropdown user user-menu">
-                                <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+                                <a href="/login/">
+                                <!-- <a href="#" class="dropdown-toggle" data-toggle="dropdown"> -->
                                     <img src="{% static 'dist/img/hip-logo.png' %}" class="user-image img-circle" alt="User Image">
-                                    <span class="hidden-xs">Hipara</span>
+                                    <span class="hidden-xs">Login</span>
                                 </a>
-                                <ul class="dropdown-menu">
+                                <!-- <ul class="dropdown-menu">
                                     <li class="user-header">
                                         <img src="{% static 'dist/img/hip-logo.png' %}" class="img-circle" alt="User Image">
                                         <p>
                                             Hipara
-                                            <!--<small>Lorem ipsum potem dorec djsh</small>-->
+                                            <small>Lorem ipsum potem dorec djsh</small>
                                         </p>
                                     </li>
                                     <li class="user-body text-center">
                                         <div class="col-xs-12 text-center">
                                             <a href="/login/">Sign in</a>
                                         </div>
-                                        <!--<div class="col-xs-6 text-center">-->
+                                        <div class="col-xs-6 text-center">-->
                                             <!--<a href="hiparaviews/register.html">Sign up</a>-->
-                                        <!--</div>-->
+                                        <!--</div>
                                     </li>
-                                </ul>
+                                </ul> -->
                             </li>
                             {% endif %}
                         </ul>
--- yaramanager-1.1.orig/source/hipara/templates/index.html
+++ yaramanager-1.1/source/hipara/templates/index.html
@@ -4,7 +4,8 @@
 {% block content %}
 
 <div class="content-wrapper">
-    <section id="home" class="info_sections3" style="background-image:url({% static page.content.top_banner.image.path %})" alt="{{page.content.top_banner.image.alt}}" title="{{page.content.top_banner.image.title}}">
+    <section id="home" class="info_sections3" alt="{{page.content.top_banner.image.alt}}" title="{{page.content.top_banner.image.title}}">
+        <!-- style="background-image:url({% static page.content.top_banner.image.path %})" -->
         <div></div>
         <div class="row">
             <div class="columns col-lg-8 col-sm-10 text-center centered">
@@ -13,6 +14,10 @@
                 </h1>
                 <h3>{{page.content.top_banner.sub_header}}</h3>
                 <p>{{page.content.top_banner.oneliner}}</p>
+
+                <iframe src="https://player.vimeo.com/video/158696349" width="500" height="284" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+                <p><a href="https://vimeo.com/158696349">hipara-intro</a> from <a href="https://vimeo.com/user49930417">J Brett Cunningham</a> on <a href="https://vimeo.com">Vimeo</a>.</p>
+                <a href="../static/downloads/deployment_guide.txt" download target="_blank"><i class="fa fa-windows"></i> Windows 32-bit Hipara</a><a href="../static/downloads/deployment_guide.txt" download target="_blank"><i class="fa fa-windows"></i> Windows 64-bit Hipara</a>
             </div>
         </div>
     </section>
@@ -61,4 +66,4 @@
         </div>
     </div>
 </div>
-{% endblock %}
\ No newline at end of file
+{% endblock %}
